python/requirements.txt

```diff
@@ -6,7 +6,7 @@ distro == 1.4
 mozdebug == 0.3
 mozinfo == 1.2.1
 mozlog == 7.1.0
-setuptools == 60.8.1
+setuptools == 65.3.0
```

Comment:
```suggestion
```

If I understand correctly, specifying setuptools in `python/requirements.txt` is not needed at all because virtualenv installs a bundled version of setuptools by default. (Assuming this file is solely used to set up a virtualenv environment.)

It may be possible that specifying it here results in a corrupted installation of setuptools. When I reproduced this issue, `servo/python/_virtualenv3.10/lib/python3.10/site-packages/setuptools` contained files from different versions of setuptools ([`_path.py`](https://github.com/pypa/setuptools/blob/v60.8.1/setuptools/_path.py) from v60.8.1 and [`config/expand.py`](https://github.com/pypa/setuptools/blob/v65.3.0/setuptools/config/expand.py#L48) from v65.3.0).